fedora/jellyfin.service

```diff
@@ -8,7 +8,7 @@ EnvironmentFile = /etc/sysconfig/jellyfin
 User = jellyfin
 Group = jellyfin
 WorkingDirectory = /var/lib/jellyfin
-ExecStart = /usr/bin/jellyfin $JELLYFIN_WEB_OPT $JELLYFIN_RESTART_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JELLYFIN_NOWEBAPP_OPT $JELLYFIN_ADDITIONAL_OPTS
```

Comment:
seems like there was removed too much here. `$JELLYFIN_FFMPEG_OPT` is not supposed to be deleted.

Reply:
You replied to a 2 year old PR for a platform that the Jellyfin team does not support (Fedora).

Reply:
oh sorry, I was browsing the mr branch instead of jellyfin master :/